### ISSUE TITLE
headless: invalid data; undefined ?

### ISSUE THREAD

**Issue by darkl1ch** [meepen/salien-bot#176]
```
Running for:      10h 14m 6s
Current level:    25 (28762727 / undefined)
Exp since start:  5633870
Estimated exp/hr: 75130
Current planet:   Under Water Exploration Planet II [16.644%] (id 531)
Current zone:     (6, 2) hard [10.629%] (id: 30)
Round time left:  54s
Next level up:    Invalid Date
```

invalid date and undefined? :\
level 26 ?

**Comment by dakkarino** [comment]
Level 25 is the maximum level you can get anyway.

https://www.reddit.com/r/Saliens/comments/8t0o1c/xp_table/

**Comment by darkl1ch** [comment]
ah. you are right.
[https://steamdb.info/badges/?badgeid=29](https://steamdb.info/badges/?badgeid=29)
somehow i remembered lvl26 there, but as far as i can see, i was wrong. :(
sill would be nice to see there MAX, or something. :)

**Comment by ChaoticHolon** [comment]
Indeed, it would be nice if the script handled this gracefully and maybe displayed N/A or MAX instead of undefined and Invalid Date. 

Or even better, when you reach 25, redefine 'Current level' to just `${info.level} (${info.score})`, hide/disable the 'Next level up' push entirely, and possibly push another field that indicates you are at max level.
